pom.xml

```diff
@@ -60,30 +60,6 @@
     </license>
   </licenses>
 
-  <modules>
-    <module>hbase-resource-bundle</module>
-    <module>hbase-server</module>
-    <module>hbase-rest</module>
-    <module>hbase-shell</module>
-    <module>hbase-protocol</module>
-    <module>hbase-client</module>
-    <module>hbase-hadoop-compat</module>
-    <module>hbase-common</module>
-    <module>hbase-procedure</module>
-    <module>hbase-it</module>
-    <module>hbase-examples</module>
-    <module>hbase-prefix-tree</module>
-    <module>hbase-assembly</module>
-    <module>hbase-testing-util</module>
-    <module>hbase-annotations</module>
-    <module>hbase-checkstyle</module>
-    <module>hbase-external-blockcache</module>
-    <module>hbase-shaded</module>
-    <module>hbase-archetypes</module>
-    <module>hbase-metrics-api</module>
-    <module>hbase-metrics</module>
-    <module>hbase-hbtop</module>
-  </modules>
```

Comment:
@Reidddddd Just to understand this better, is it possible to keep modules here as is and then override in specific profile if required? Like we continue to have this and then JDK7 can override this, so that default profile would not have to define <modules> again.
The problem is, IDEs might not be able to identify these modules by parsing pom.xml. Thoughts?